docs/ppl/commands.md

```diff
@@ -617,7 +590,7 @@ search source=accounts | rare age by gender;
 | M  | 32
 | M  | 33
 
-## top {#top-command}
```

Comment:
Hi @chloe-zh, this actually replaces the auto-generated ID of `#top` with `#top-command` so that the righthand navigation takes you to the proper section rather than the top of the webpage. It's worth keeping.

Reply:
Ok I see, sounds good I'll add it back, thanks!